src/uct/base/uct_iface.c

```diff
@@ -548,3 +549,42 @@ ucs_config_field_t uct_iface_config_table[] = {
 
   {NULL}
 };
+
+ucs_status_t uct_base_ep_am_short_iov(uct_ep_h ep, uint8_t id, const uct_iov_t *iov,
+                                      size_t iovcnt)
+{
+    unsigned length;
+    void *buffer;
+    size_t iov_it;
+    uint64_t header;
+    void *payload;
+    ucs_status_t status;
+
+    length = uct_iov_total_length(iov, iovcnt);
+    if (length > 1200) {
+        buffer = malloc(length);
+    } else {
+        buffer = ucs_alloca(length);
+    }
+    length = 0;
+
+    for (iov_it = 0; iov_it < iovcnt; ++iov_it) {
```

Comment:
`ucs_iov_copy(iov, iovcnt, 0, buffer, length , UCS_IOV_COPY_TO_BUF);` instead of the loop

Reply:
This helper method is for iovec.